### ISSUE TITLE
Clients cannot distinguish entity to display in types

### ISSUE THREAD

**Issue by vanaukenk** [geneontology/noctua#638]
I'm not sure if this is a minerva issue or something else, but in a test model, the C. elegans gene pmk-1, is displayed in the table as 'information biomacromolecule' instead of pmk-1.

The label for another gene in the table, skn-1, displays fine.

http://noctua-dev.berkeleybop.org/workbench/noctua-form/?model_id=gomodel:5dd87d1500000000

![image](https://user-images.githubusercontent.com/1730534/69556483-c724fd80-0f72-11ea-92c1-ab48d33bacd2.png)

**Comment by kltm** [comment]
Moving as this is a more general noctua issue.

**Comment by goodb** [comment]
Yes.  its behaving as designed from the no-neo in go-lego server perspective.  I believe the client should decide what type to use for the display.  Note this issue only occurs for genes and the only other types it will show are information biomacromolecule or protein.

**Comment by kltm** [comment]
Let's work through how the clients operate with Minerva.

Using the example above, let's say I decide that I've made an error with sek-1 and I change it to something that is a protein, instead of an information biomacromolecule: a delete and add operation of a single class expression. Will Minerva automatically change the second class expression to protein? If not, we may want to either move this information to inferred type or figure out some other way of storing/updating it.

**Comment by kltm** [comment]
@vanaukenk How disruptive is this while we figure out what to do? Should we revert the code on the production server to the last state?

**Comment by vanaukenk** [comment]
@kltm The main issue is that in the table view of the Noctua form, the gene name isn't consistently displayed, so if a curator wants to perform any operations starting from the table, they don't know what gene(s) the 'information biomacromolecule' refers to, so for the form it is a showstopper.

For the graph editor, the extra label might add some clutter to the display, but all of the information needed is there, so here it's more of a visualization issue than a curation workflow issue.

**Comment by kltm** [comment]
@vanaukenk A showstopper for now then. I'll announce a maintenance window for tomorrow, giving people a chance to save, then roll it back in the afternoon PST. We can revisit after the break.

**Comment by vanaukenk** [comment]
Okay, thanks @kltm

**Comment by kltm** [comment]
Temporarily restarted minerva at https://github.com/geneontology/minerva/commit/5ae8bf16e752327eda1d8671997cad4f34cb838b

Noting (from https://github.com/geneontology/noctua-models/issues/133) that http://noctua.berkeleybop.org/editor/graph/gomodel:5dd88cab00000142 still has the additions.

**Comment by kltm** [comment]
Talking a little about what's going on here with @cmungall . We'll likely hold on the earlier version of minerva for a little bit while the options are discussed (hopefully in some carved out time next week, but we'll get word).

**Comment by kltm** [comment]
In the interim, we may need to clean up/out (manually if only a few) models edited in the last few days.
